templates/defaults.cfg

```diff
@@ -49,7 +49,7 @@ defaults
     option {{ option }}
 {% endfor -%}
 {% endif -%}
-{% if ansible_distribution != 'CentOS' %}
+{% if ansible_distribution != 'CentOS' and ansible_distribution != 'Alpine' %}
```

Comment:
I see, thanks. As-is this means users can't actually override `haproxy_defaults` and set error files if they install the files out-of-band.

We need to provide the user with a mechanism for configuring the files if they want to on these distributions. That's out of scope for this PR, but I'll log an issue for it.